frontend/dockerui/build.go

```diff
@@ -133,3 +126,19 @@ func extendWindowsPlatform(p, imgP ocispecs.Platform) ocispecs.Platform {
 	}
 	return p
 }
+
+func normalizePlatform(p, imgP ocispecs.Platform) exptypes.Platform {
+	var k string
+	if p.OS == "windows" {
+		k = platforms.FormatAll(p)
```

Comment:
Shouldn't this be normalized as well before formatting?

Reply:
Didn't change the logic for Windows, as I don't have an env to test the changes